docs/api_call.md

```diff
@@ -1,1352 +1,1108 @@
-      URL='localhost'
-      PORT=8081
-      ```
-
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/text/?message=i%20want%20to%20order%20chinese%20from%20%20mainland%20china%20and%20pizza%20from%20domminos&entity_name=restaurant&structured_value=&fallback_value=None&bot_message='
-      ```
-
-    - *CURL Output:*
-
-      ```json
-      {
-        "data": [
-          {
-            "detection": "message",
-            "original_text": "mainland china",
-            "entity_value": {
-              "value": "Mainland China"
-            }
-          },
-          {
-            "detection": "message",
-            "original_text": "dominos",
-            "entity_value": {
-              "value": "Domino's Pizza"
-            }
-          }
-        ]
-      }
-      ```
-
-  - Example 2: 
-
-    - ```python
-      message = 'i wanted to watch movie'
-      entity_name = 'movie'
-      structured_value = 'inferno'
-      fallback_value = None
-      bot_message = None
-      ```
-
-    - *Python:* 
-
-      ```python
-      from ner_v1.chatbot.entity_detection import get_text
-      output = get_text(message=message, entity_name=entity_name, structured_value=structured_value, fallback_value=fallback_value, bot_message=bot_message)
-      print output
-      ```
-
-    - *CURL command:*
-
-      ```shell
-      URL='localhost'
-      PORT=8081
-      ```
-
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/text/?message=i%20wanted%20to%20watch%20movie&entity_name=movie&structured_value=inferno&fallback_value=None&bot_message='
-      ```
-
-    - *CURL Output*:
-
-    - ​
-
-      ```json
-      {
-        "data": [
-          {
-            "detection": "structure_value_verified",
-            "original_text": "inferno",
-            "entity_value": {
-              "value": "Inferno"
-            }
-          }
-        ]
-      }
-      ```
-
-  - ​
-
-
-### phone_number
+2. **entity_name**: Name of the entity. This parameter is important when the detection logic relies on dictionary. It is used to run particular detection logic by looking at its dictionary.
 
-- This functionality calls the PhoneDetector class to detect textual entities.
+   For example, in Text detection logic  the entity name is based on dictionary name of entities we need to detect.
 
-- Example:
+   - For detecting:
+     - *cuisine* entity_name will be *"cuisine"*
+     - *dish* entity_name will be *"dish"*
 
-  - Example 1: 
+   In some detection like `date` or `time detection` `entity_name` does not have any siginificance as detection logic of these entities won't change based on their name.
 
-    - ```python
-      message = 'my contact number is 9049961794'
-      entity_name = 'phone_number'
-      structured_value = None
-      fallback_value = None
-      bot_message = None
-      ```
-
-    - *Python:* 
+3. **structured_value**: It is a value which is obtained from the structured text. For example, UI elements like form, payload, etc.
```

Comment:
This needs more detailed and better explanation. Sentence "For example, UI elements like form, payload, etc." is vague without any background of chatbot. Try to upload a picture of form from haptik and facebook messenger

Reply:
Should we even explain `structured_value` and `fallback_value`. I feel like it is too much information and might even cause confusion unless we explain what exactly goes behind the scene. When structured value will be considered, what structured value not verified means, when is fallback value is returned as it is and when is it parsed into output format, etc

Reply:
Please decide and let me know. According to me if we have mentioned, this is optional, then it is not extra information for them.